routing/routes.go

```diff
@@ -25,27 +25,85 @@ func NewRoutes() (r *Routes) {
 	r = new(Routes)
 
 	r.Routes = []Route{
+		// swagger:route GET / Root
+		// ---
+		// Handler to get the status of Kiali
```

Comment:
If this is going to the documentation, I would use "Endpoint" rather than "Handler".
As end user Endpoint is clear, "handler" is more the terminology used to implement it (I think).

Reply:
"Handler" is used in several locations, I won't mark them all of them to avoid noise.